spring-cloud-loadbalancer/src/main/java/org/springframework/cloud/loadbalancer/core/LoadBalancerServiceInstanceCookieTransformer.java

```diff
@@ -14,12 +14,15 @@
  * limitations under the License.
  */
 
-package org.springframework.cloud.client.loadbalancer;
+package org.springframework.cloud.loadbalancer.core;
```

Comment:
This is a breaking change, but was this just introduced for this release?

Reply:
This class has not been released yet :).